go.mod

```diff
@@ -227,10 +227,10 @@ require (
 
 require (
 	cloud.google.com/go/kms v1.10.1 // @grafana/backend-platform
-	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.2.0 // @grafana/backend-platform
-	github.com/Azure/azure-sdk-for-go/sdk/keyvault/azkeys v0.9.0 // @grafana/backend-platform
-	github.com/Azure/azure-storage-blob-go v0.15.0 // @grafana/backend-platform
-	github.com/Azure/go-autorest/autorest/adal v0.9.22 // @grafana/backend-platform
+	github.com/Azure/azure-sdk-for-go/sdk/azidentity v1.2.0 // @grafana-partner-datasources
+	github.com/Azure/azure-sdk-for-go/sdk/keyvault/azkeys v0.9.0 // @grafana-partner-datasources
+	github.com/Azure/azure-storage-blob-go v0.15.0 // @grafana-partner-datasources
```

Comment:
This one does not belong to us, I believe this is backend-platform as it's part of the image uploader.